src/main/java/com/cflint/plugins/core/FunctionXChecker.java

```diff
@@ -0,0 +1,35 @@
+package com.cflint.plugins.core;
+
+import ro.fortsoft.pf4j.Extension;
+import net.htmlparser.jericho.Element;
+import cfml.parsing.cfscript.CFExpression;
+import cfml.parsing.cfscript.CFFunctionExpression;
+import cfml.parsing.cfscript.script.CFExpressionStatement;
+import cfml.parsing.cfscript.script.CFScriptStatement;
+
+import com.cflint.BugInfo;
+import com.cflint.BugList;
+import com.cflint.plugins.CFLintScannerAdapter;
+import com.cflint.plugins.Context;
+
+@Extension
+public class FunctionXChecker extends CFLintScannerAdapter {
+	final String severity = "INFO";
+
+	@Override
+	public void expression(CFExpression expression, Context context,
+			BugList bugs) {
+		if (expression instanceof CFFunctionExpression) {
+			final String cfmlFunctionCheck = getParameter("functionName");
+
+			CFFunctionExpression functionExpression = (CFFunctionExpression)expression;
+			if(functionExpression.getName().equalsIgnoreCase(cfmlFunctionCheck)){
```

Comment:
may want to use cfmlFunctionCheck  + "(" so less chance of matching things that are not function calls?